TradeTrust/tradetrust-website
# feat: adding validation when saving endpoint

## 3rd Party Resolver Validation Upon Save

Story: https://www.pivotaltracker.com/story/show/173395984

Allow the client to validate the endpoint's auth and features upon saving the endpoint. This will also pave way for allowing different features on the endpoint to be turned on or off in the future.

Also fixes the case where subsequent endpoints were never queried.

### To Test

![image](https://user-images.githubusercontent.com/7406870/91926755-841a6680-ed0a-11ea-83b3-7ed94cea72d2.png)

1. Enter the root url of the demo resolver and it's corresponding header and key and it should save.
2. Try it with and without trailing slashes
3. Try it with wrong header/key
